RSI

```diff
@@ -15,8 +13,21 @@ start_time = datetime.datetime(2020, 10, 1)
 #end_time = datetime.datetime(2020, 1, 20)
 end_time = datetime.datetime.now().date().isoformat()         # today
 
-df = ticker_df
+connected = False
+while not connected:
+    try:
+        ticker_df = web.get_data_yahoo(ticker, start=start_time, end=end_time)
+        connected = True
+        print('connected to yahoo')
+    except Exception as e:
+        print("type error: " + str(e))
+        time.sleep( 5 )
+        pass
 
+ticker_df = ticker_df.reset_index()
+print(ticker_df.head(5))
+
+df = ticker_df
```

Comment:
Is it necessary to create 'df' if 'ticker_df' is exactly the same?

Reply:
not really, but I place it in case i switch out the ticker_df

Reply:
I'd suggest deleting this declaration if there's no purpose.